Sources/SnowManLogic/GameLogic/guessManager.swift

```diff
@@ -0,0 +1,24 @@
+public class GuessManager {
+	public var totalIncorrectGuessesAllowed: Int
+	public var correctGuesses: [String] = []
+	public var incorrectGuesses: [String] = []
+
+	public init (totalIncorrectGuessesAllowed: Int) {
+		self.totalIncorrectGuessesAllowed = totalIncorrectGuessesAllowed
+	}
+
+	public func findUnguessedLetters(letters: [String]) -> Set<String> {
```

Comment:
`find` often suggests that some sort of query is happening.  I suggest changing this to `unguessedLetters()`